modules/music-release-cc.js

```diff
@@ -0,0 +1,78 @@
+var markdown = require('ssb-markdown');
+var h = require('hyperscript');
+var u = require('../util');
+var ref = require('ssb-ref');
+
+//render a message
+
+var plugs = require('../plugs');
+var message_link = plugs.first(exports.message_link = []);
+var message_confirm = plugs.first(exports.message_confirm = []);
+var sbot_links = plugs.first(exports.sbot_links = []);
+
+exports.message_content = function(msg, sbot) {
+    if (msg.value.content.type !== 'music-release-cc')
+        return;
+
+    var tracks = msg.value.content.tracks;
+    return h('div',
+        h('img', { "src" : "http://localhost:7777/" + encodeURIComponent(msg.value.content.cover) }),
+        h('h1', msg.value.content.title),
+        h('ol',
+                 Object.keys(tracks).map(function(k) {
+                     var t = tracks[k];
+                     return h('li', t.fname,
+                         h("br"),
+                         h('audio', {
+                                  "controls" : true,
+                                  "src" : "http://localhost:7777/" + encodeURIComponent(t.link)
+                              }))
+                 })),
+        h('p',
+                 "More info:", h('a', { href : msg.value.content.archivedotorg }, "archive.org"),
```

Comment:
This was really just prototyping and a way to attribute the source. 

How would you edit these, though? It's also why I shunned away from the fat objects...
We didn't even touch song metadata yet.